api/metrics/v1/http.go

```diff
@@ -30,6 +32,7 @@ const (
 	labelNamesRoute  = "/api/v1/labels"
 	labelValuesRoute = "/api/v1/label/{label_name}/values"
 	receiveRoute     = "/api/v1/receive"
+	rulesRoute       = "/api/v1/rulesfile"
```

Comment:
naming is hard :see_no_evil: `/api/v1/rulesfile` sounds a bit funny to me though since when the user `put`s to the API, you're not really creating a file. hmm maybe `/api/v1/raw_rules`?

Reply:
it is indeed :sweat_smile: we thought `rulesfile` because the content has the same format as the Prometheus rules file - but I agree that gets confusing.. I'm fine with `raw_rules` too @onprem what do you think?

Reply:
Yep that sounds good. Another one I can think of from what Lucas suggested is `/api/v1/rules/raw`, which just feels a bit more natural to me. WDYT @squat @jessicalins ?

Reply:
^ I like it :) aligns better with the other rules endpoints too

Reply:
I second this